### PR TITLE
BM Admin Adjustments: creation flow

### PR DESCRIPTION
**Pull request by yurii-lubynets**
closes sparkletown/internal-sparkle-issues#959
closes https://github.com/sparkletown/internal-sparkle-issues/issues/958
closes https://github.com/sparkletown/internal-sparkle-issues/issues/957

### CODE REVIEW

File: src/pages/Admin/Venue/DetailsForm.tsx [hunk 1]
```diff
@@ -176,52 +148,14 @@ export const DetailsForm: React.FC<DetailsFormProps> = ({
...
-                );
-              }
-            });
         } else
           await createVenue(
             {
               ...vals,
-              userStatuses,
-              showUserStatus: showUserStatuses,
+              parentId: "/playa",
```
Comment (alexey-petrov) [inline src/pages/Admin/Venue/DetailsForm.tsx:158]: can we have it moved to constants somewhere? settings.ts for example

Comment (yurii-lubynets) [reply src/pages/Admin/Venue/DetailsForm.tsx]: fixed in 14efccebfb3354121951a37f42bc254dcad58ad5

**Comment by denisdimitrov** [comment]
@yurii-lubynets This PR looks almost identical to https://github.com/sparkletown/sparkle/pull/1930, why not add those changes here and leave only a single PR?

**Comment by yurii-lubynets** [comment]
@denisdimitrov you are right. I've added all tiny changes into https://github.com/sparkletown/sparkle/pull/1930 and closed this one

**Pull request closed without merging**